pkg/pillar/types/contenttreetypes.go

```diff
@@ -188,6 +194,9 @@ func (status ContentTreeStatus) LogDelete() {
 		status.ContentID, status.LogKey())
 	logObject.CloneAndAddField("contentSha256", status.ContentSha256).
 		AddField("maxDownloadSize", status.MaxDownloadSize).
+		AddField("state", status.State.String()).
+		AddField("progress", status.Progress).
```

Comment:
Small nit: Add ```int64``` in the string.

Reply:
Oh, it is updated in the next commit. I didn't see that.

Reply:
progress is old, and as such is known to be an int64.

Reply:
OK